planner/aamwriter/src/main/resources/examples/drafts/simple_dam.yaml

```diff
@@ -0,0 +1,48 @@
+

```

Comment:
No need to add this, because the DAM looks differently, I can provide a full DAM example if needed.